### PR TITLE
Fix helm install commands in docs

### PR DESCRIPTION
**Pull request by muvaf**
### Description of your changes

Helm 2 doesn't have `--devel` flag and Helm3 requires `--disable-openapi-validation` in older Kubernetes versions.

### How has this code been tested?

Manually.

### Checklist

I have:
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Ensured this PR contains a neat, self documenting set of commits.
- [x] Updated any relevant [documentation] and [examples].
- [x] Reported all new error conditions into the log or as an event, as
  appropriate.

For more about what we believe makes a pull request complete, see our
[definition of done].

[documentation]: https://github.com/crossplane/crossplane/tree/master/docs
[examples]: https://github.com/crossplane/crossplane/tree/master/cluster/examples
[definition of done]: https://github.com/crossplane/crossplane/tree/master/design/one-pager-definition-of-done.md

### CODE REVIEW

File: cluster/charts/crossplane/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Install with Helm 2:
 
 ```console
 helm repo add crossplane-master https://charts.crossplane.io/master/
-helm search crossplane-master --devel
```
Comment (hasheddan) [inline cluster/charts/crossplane/README.md:64]: I am fairly certain more recent versions of helm 2 do include `--devel`. I recently updated the integration tests to use `--devel` and I believe they use helm 2.

Comment (hasheddan) [reply cluster/charts/crossplane/README.md:64]: If you do not specify it then it uses `alpha`

Comment (hasheddan) [reply cluster/charts/crossplane/README.md:64]: https://v2.helm.sh/docs/helm/#options-8

Comment (muvaf) [reply cluster/charts/crossplane/README.md:64]: > If you do not specify it then it uses alpha

I think that happens when you run `helm search crossplane` but here we're explicitly looking in `crossplane-master`. Running helm 2.15.1 `helm search crossplane-master` gives me:
```
NAME                                    	CHART VERSION         	APP VERSION           	DESCRIPTION
crossplane-master/crossplane            	0.11.0-rc.146.g7d91391	0.11.0-rc.146.g7d91391	Crossplane is an open source Kubernetes add-on that exten...
crossplane-master/crossplane-controllers	0.11.0-rc.146.g7d91391	0.11.0-rc.146.g7d91391	Crossplane - Managed Cloud Resources Operator
crossplane-master/crossplane-types      	0.11.0-rc.146.g7d91391	0.11.0-rc.146.g7d91391	Crossplane - Managed Cloud Resources Operator
crossplane-master/gitlab-controller     	0.0.0-43.90e2a0e      	0.0.0-43.90e2a0e      	GitLab Controller - GitLab Crossplane Application
```

Comment (hasheddan) [reply cluster/charts/crossplane/README.md:64]: Gotcha! Thanks for clarifying :)

**Pull request merged**